File: app-service-comments/src/test/java/com/github/arhor/aws/graphql/federation/comments/test/ConfigureTestObjectMapper.java
package com.github.arhor.aws.graphql.federation.comments.test;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.json.JsonMapper;
import org.springframework.boot.test.context.TestConfiguration;
import org.springframework.context.annotation.Bean;

@TestConfiguration(proxyBeanMethods = false)
public class ConfigureTestObjectMapper {

    @Bean
    public ObjectMapper objectMapper() {
        return JsonMapper.builder().build();
    }
}

File: app-service-posts/src/test/kotlin/com/github/arhor/aws/graphql/federation/posts/test/ConfigureTestObjectMapper.kt
package com.github.arhor.aws.graphql.federation.posts.test

import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
import org.springframework.boot.test.context.TestConfiguration
import org.springframework.context.annotation.Bean

@TestConfiguration(proxyBeanMethods = false)
class ConfigureTestObjectMapper {

    @Bean
    fun objectMapper() = jacksonObjectMapper()
}
